# request
communication_software/Message command
---------------
# result
uint8 SUCCESS = 0
uint8 INVALID_COMMAND = 1
uint8 SEND_FAILED = 2

uint8 result
string message

